README.md

```diff
@@ -104,6 +104,8 @@ After you're finished please remove all the comments and instructions!
 
```

Comment:
![calculator](https://user-images.githubusercontent.com/97521557/211631803-4e583bf3-c3d5-4dfe-ae4a-b878d86d3796.png)

**[Optional]**

- [ ] I guess you misunderstood previous reviewer suggested change :smile: becuase this black border is not a part of your calculator design. It would be nice if you remove this black border from your calculator :+1: